lighthouse-core/lib/page-functions.js

```diff
@@ -107,12 +107,20 @@ function getElementsInDocument(selector) {
 /**
  * Gets the opening tag text of the given node.
```

Comment:
update the jsdoc string? Something like "Optionally provide an array of attribute names to not include in the tag snippet." (or whatever)